proto/babylon/finality/v1/tx.proto

```diff
@@ -103,3 +107,18 @@ message MsgUnjailFinalityProvider {
 
 // MsgUnjailFinalityProviderResponse defines the Msg/UnjailFinalityProvider response type
 message MsgUnjailFinalityProviderResponse {}
+
+// MsgEquivocationEvidence is the message for handling evidence of equivocation
+message MsgEquivocationEvidence {
+    option (cosmos.msg.v1.signer) = "signer";
+
+    string signer = 1;
+    // evidence is the evidence of equivocation
+    Evidence evidence = 2 [(gogoproto.nullable) = false];
```

Comment:
Suggest we have all fields in `Evidence` here rather than using a `Evidence` object directly. This is because `Evidence` is a DB object and it's better not conflate DB objects and message objects

Reply:
`Evidence` is defined in [here](https://github.com/babylonlabs-io/babylon/blob/base/consumer-chain-support/proto/babylon/finality/v1/finality.proto#L92-L113), so it is a message object, right?